packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -1,21 +1,27 @@
 <template>
-	<button
-		:class="mainClasses"
-		role="button"
-		:type="isSubmit ? 'submit' : 'button'"
-		@click="handleClick"
-	>
-		<i v-if="isLoading" class="kt-circle-loading" />
-		<i v-else-if="icon !== null" class="yoco" v-text="icon" />
-		<span v-if="hasSlot">
-			<slot />
-		</span>
-		<span v-else-if="label !== null" v-text="label" />
-	</button>
+	<div>
+		<button
+			:class="mainClasses"
+			ref="triggerRef"
```

Comment:
```suggestion
			ref="triggerRef"
			:class="mainClasses"
```

Reply:
@Juli0GT we need to reconfigure the liner on your machine :D

Locally please run `yarn run fix` for now